src/families/stellar/api/horizon.js

```diff
@@ -0,0 +1,266 @@
+//@flow
+import { BigNumber } from "bignumber.js";
+import StellarSdk from "stellar-sdk";
+import { getEnv } from "../../../env";
+import {
+  formatCurrencyUnit,
+  getCryptoCurrencyById,
+  parseCurrencyUnit,
+} from "../../../currencies";
+import type { Account, NetworkInfo, Operation } from "../../../types";
+import type { RawAccount, RawTransaction } from "./horizon.types";
+import { getAccountSpendableBalance, rawOperationToOperation } from "../logic";
+
+const LIMIT = 200;
+const FALLBACK_BASE_FEE = 100;
+
+const currency = getCryptoCurrencyById("stellar");
+
+const baseAPIUrl = getEnv("API_STELLAR_HORIZON");
+const server = new StellarSdk.Server(baseAPIUrl);
```

Comment:
it's a low but I think we could make this:

- lazy
- handling the possible changes of API_STELLAR_HORIZON env over time (e.g. experimental settings)